sdk/program/src/vote/state/mod.rs

```diff
@@ -827,7 +862,7 @@ impl VoteState {
         if self.epoch_credits.is_empty() {
             0
         } else {
-            self.epoch_credits.last().unwrap().0
+            self.epoch_credits.last().unwrap().epoch
```

Comment:
nit: doesn't have to happen in this PR but this is
self.epoch_credits.last().unwrap_or(0) (same below for credits())